gossip/filters/filter_test.go

```diff
@@ -55,7 +54,7 @@ func makeReceipt(addr common.Address) *types.Receipt {
 }
 
 func BenchmarkFilters(b *testing.B) {
-	dir, err := ioutil.TempDir("", "filtertest")
+	dir, err := os.MkdirTemp("", "filtertest")
```

Comment:
How about using [b.TempDir()](https://pkg.go.dev/testing#B.TempDir) at this place? This would be a directory that is unique and automatically cleaned up at the end of the benchmark.

Reply:
You are right, for testing is even better to let the test tool handle temporaries